synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -81,29 +126,38 @@ def lint_setup_py(session):
     session.run("python", "setup.py", "check", "--restructuredtext", "--strict")
 
 
+def install_unittest_dependencies(session, *constraints):
+    session.install(*UNIT_TEST_STANDARD_DEPENDENCIES, *constraints)
+
+    if UNIT_TEST_EXTERNAL_DEPENDENCIES:
+        session.install(*UNIT_TEST_EXTERNAL_DEPENDENCIES, *constraints)
+
+    if UNIT_TEST_LOCAL_DEPENDENCIES:
+        session.install("-e", *UNIT_TEST_LOCAL_DEPENDENCIES, *constraints)
+
+    if UNIT_TEST_DEPENDENCIES:
+        session.install("-e", *UNIT_TEST_DEPENDENCIES, *constraints)
```

Comment:
These aren't supposed to be installed in "editable" mode, are they?

Reply:
Likely not -- I doubt anybody is using a `git:` URL for a dependency.

Reply:
d274ce3